### DISCUSSION TITLE
I don't know why the PR I raised needs to be approved to execute UT (I'm not a first-time contributor). Can someone please  help me find out why UT approval is needed. Thanks.

### DISCUSSION THREAD

**Discussion by hililiwei** [community/community#35679] [Pull Requests]
I don't know why the PR I raised needs to be approved to execute UT (I'm not a first-time contributor). Can someone please  help me find out why UT approval is needed. Thanks.
For example, the following PR
https://github.com/apache/iceberg/pull/5029
https://github.com/apache/iceberg/pull/5942

**Comment by MitkoVtori** [comment]
It's because you're trying to pull changes from your branch(or something) to the master branch. The Owner of the branch have set rules so you can't merge without approve.
On my Repo I also have such a think:
https://github.com/MitkoVtori/Mitko.Vtori-World/pull/13
![Screenshot 2022-10-11 145310](https://user-images.githubusercontent.com/112943652/195082767-ea632f15-8060-4ffe-9275-d9525195167b.png)
You should ask the owner of the repo to go to the branch, click on master branch rules, and edit them.

**Reply by HackerstoyanovBG** [reply]
It worked with me. So I think it should work with him.
